pkg/storage/stores.go

```diff
@@ -447,7 +447,7 @@ func SynthesizeClusterVersionFromEngines(
 		// restarting into 1.1 after having upgraded to 1.2 doesn't work.
 		for _, v := range []roachpb.Version{cv.MinimumVersion, cv.UseVersion} {
 			if serverVersion.Less(v) {
-				return cluster.ClusterVersion{}, errors.Errorf("engine %s requires at least v%s, but running version is %s",
+				return cluster.ClusterVersion{}, errors.Errorf("store %s requires at least cockroach version v%s, but running version is v%s",
```

Comment:
This is much clearer to me. Thanks, @a-robinson! The only nit I have is that `running version` is perhaps inaccurate in that cockroach won't actually run in this case. Would something like this work?

```
"cockroach version v%s is incompatible with data in store %s; use version v%s or later"
```

Reply:
Sure, done.